database/tables/load_bond_db.sql

```diff
@@ -0,0 +1,148 @@
+
+-- Create custom types
+
+CREATE TYPE AssetType AS ENUM ('CURRENCY', 'BOND');
+CREATE TYPE QuoteStatus AS ENUM ('OPEN', 'CLOSED');
+CREATE TYPE OrderStatus AS ENUM ('OPEN', 'MATCHED', 'SETTLED');
+CREATE TYPE OrderType AS ENUM ('MARKET', 'LIMIT');
+CREATE TYPE OrderAction AS ENUM ('BUY', 'SELL');
+CREATE TYPE OrganizationType AS ENUM ('TRADING_FIRM', 'PRICING_SOURCE');
+CREATE TYPE Role AS ENUM ('MARKET_MAKER', 'TRADER');
+CREATE TYPE PaymentType AS ENUM ('COUPON', 'REDEMPTION');
+CREATE TYPE CouponType AS ENUM ('FIXED', 'FLOATING');
+CREATE TYPE CouponFrequency AS ENUM ('QUARTERLY', 'MONTHLY', 'DAILY');
+
+
+-- Create tables
+
+CREATE TABLE IF NOT EXISTS bonds (
+  id                        bigserial   PRIMARY KEY,
+  bond_id                   varchar,
+  issuing_organization_id   varchar,
+  amount_outstanding        bigint,
+  coupon_rate               bigint,
+  first_coupon_date         date,
+  first_settlement_date     date,
+  maturity_date             date,
+  face_value                bigint,
+  coupon_type               CouponType,
+  coupon_frequency          CouponFrequency,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS corporate_dept_rates (
+  id                        bigserial   PRIMARY KEY,
+  bond_id                   varchar,
+  agency                    varchar,
+  rating                    varchar,
+  start_block_num           bigint,
+  end_block_num             bigint
+);
+
+CREATE TABLE IF NOT EXISTS organizations (
+  id                        bigserial   PRIMARY KEY,
+  organization_id           varchar,
+  industry                  varchar,
+  name                      varchar,
+  organization_type         OrganizationType,
+  pricing_source            varchar,
```

Comment:
Based on the discussion yesterday, we are rolling this field in to `organization_id`